packages/cashier/src/modules/cashier-onboarding/components/cashier-onboarding-card/cashier-onboarding-card.scss

```diff
@@ -0,0 +1,27 @@
+.cashier-onboarding-card {
+    margin-top: 2.4rem;
+
+    &__container {
+        display: flex;
+        flex-direction: column;
+        padding: 1.6rem;
+        border: 2px solid var(--border-normal-1);
+        border-radius: $BORDER_RADIUS * 2;
+        margin-top: 0.8rem;
+        cursor: pointer;
+        transition: all 0.25s ease;
+
+        &:hover {
+            transform: scale(0.98);
+        }
```

Comment:
It's not hard to add micro-interactions with couple of CSS lines, But we don't really have much through the website and I'm not sure why 🤔 

Anyway, Should I remove this?

Reply:
This looks nice, actually 🔥